dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/ResourceMap.cpp

```diff
@@ -40,15 +40,25 @@ Resources::ResourceMap ResourceMap::GetSubtree(hstring const& reference)
     return winrt::make<ResourceMap>(m_resourceManager, m_resourceManagerHandle, subtree);
 }
 
-Resources::ResourceCandidate ResourceMap::GetValueImpl(const Resources::ResourceContext* context, hstring const& resource)
+Resources::ResourceCandidate ResourceMap::GetValueImpl(const Resources::ResourceContext* context, hstring const& resource, bool notFoundOk)
 {
```

Comment:
`notFoundOK`

Odd name (negative names lead to double-negative code and harder to grok and mainain)

Suggestion: `notFoundIsOK` or `treatNotFoundAsOK`

Reply:
`allowMissing` is another, or to flip the meaning, `treatMissingAsError`.